packages/main/src/themes/BusyIndicator.css

```diff
@@ -89,6 +96,11 @@
 	animation-delay: 400ms;
 }
 
+.ui5-busyindicator-text {
+	width: 100%;
+    text-align: center;
```

Comment:
at least in github the text-align looks indented too much

Reply:
In IDE looks good